### DISCUSSION TITLE
Help an idiot to run Automatic1111 on Paperspace

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#2150] [Q&A]
While I wait in vain for Google to say something about SD I signed up on Paperspace but I can't load the repo on my notebook for the life of me. What is the URL I should paste on Workspace URL? When I try to run the colab I get this error: "File Load Error for PPS-A1111.ipynb
Failed to fetch". I also wish to know if I can set a path to a model/folder from my Google Drive account.

**Comment by TheLastBen** [comment]
https://console.paperspace.com/github/TheLastBen/PPS?machine=Free-GPU
then go back to the menu gradient and choose a different gpu

**Reply by ClaraSanders** [reply]
I was only able to run your repos by clicking on PPSS-A1111.ipynb for example and then Run instead of Start Machine. The problem is that I can't run them in private using this (I'm a paid user). Is there any real difference between public and private? Can people access it and run it instead of me or something? My only problem now is uploading models and such because Paperspace is PAINFULLY slow. I read on the help section that you can download files or folders from Drive but I don't know how to do it. 
gdown https://drive.google.com/drive/folders/<file_ID> -O /tmp/folder --folder
I changed the file_ID by my public link folder and yet I get syntax error.

**Reply by TheLastBen** [reply]
simply share your gdrive folder, copy its link, then, in a new cell in paperspace, do `!gdown --folder linktoyourfolder`
